automation/src/test/scala/org/broadinstitute/dsde/firecloud/component/Collapse.scala

```diff
@@ -5,37 +5,41 @@ import org.openqa.selenium.WebDriver
 
 case class Collapse[A <: FireCloudView](queryString: QueryString, private val inner: A)(implicit webDriver: WebDriver)
   extends Component(queryString) with Stateful {
-  private val toggleComponent = findInner("toggle")
+
+  private val toggleCssQuery: CssSelectorQuery = findInner("toggle")
+  private val toggleLink = Link(CSSQuery(toggleCssQuery.queryString))
```

Comment:
Do `private val toggleComponent = new Link("toggle" inside this) with Stateful`. Then...